error-prone-contrib/src/test/java/tech/picnic/errorprone/bugpatterns/IdentityConversionTest.java

```diff
@@ -157,6 +163,11 @@ void identification() {
             "    Mono<Integer> m1 = Mono.from(Mono.just(1));",
             "    // BUG: Diagnostic contains:",
             "    Mono<Integer> m2 = Mono.fromDirect(Mono.just(1));",
+            "",
+            "    // BUG: Diagnostic contains:",
+            "    Matcher allOfMatcher = Matchers.allOf(annotations(AT_LEAST_ONE, isType(\"foo\")));",
```

Comment:
Let's rename them to `matcher1` and `matcher2` to be in the "same style" as the other variable names.